pkg/utils/validations_test.go

```diff
@@ -0,0 +1,110 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/kubescape/k8s-interface/instanceidhandler/v1/helpers"
+)
+
+func TestValidateStatusAnnotation(t *testing.T) {
+	tests := []struct {
+		name        string
+		annotations map[string]string
+		wantErr     bool
+	}{
+		{
+			name:        "valid status - initializing",
+			annotations: map[string]string{"kubescape.io/status": "initializing"},
```

Comment:
I wonder if we don't already have consts at least for the annotation key, but also for values

Reply:
We are using consts all around the code..
in the tests I prefer to use hardcoded strings so that it breaks in case of changes...